src/twisted/internet/task.py

```diff
@@ -40,39 +55,37 @@ class LoopingCall:
         something else, but it probably ought to be set *before*
         calling L{start}.
 
-    @type running: C{bool}
     @ivar running: A flag which is C{True} while C{f} is scheduled to be called
         (or is currently being called). It is set to C{True} when L{start} is
         called and set to C{False} when L{stop} is called or if C{f} raises an
         exception. In either case, it will be C{False} by the time the
         C{Deferred} returned by L{start} fires its callback or errback.
 
-    @type _realLastTime: C{float}
     @ivar _realLastTime: When counting skips, the time at which the skip
         counter was last invoked.
 
-    @type _runAtStart: C{bool}
     @ivar _runAtStart: A flag indicating whether the 'now' argument was passed
         to L{LoopingCall.start}.
     """
 
-    call = None
+    call: Optional[IDelayedCall] = None
     running = False
-    _deferred = None
-    interval = None
+    _deferred: Optional[Deferred["LoopingCall"]] = None
+    interval: Optional[float] = None
     _runAtStart = False
-    starttime = None
+    starttime: Optional[float] = None
+    _realLastTime: Optional[float] = None
 
-    def __init__(self, f, *a, **kw):
+    def __init__(self, f: Callable[..., object], *a: object, **kw: object) -> None:
         self.f = f
         self.a = a
         self.kw = kw
         from twisted.internet import reactor
 
-        self.clock = reactor
+        self.clock: IReactorTime = cast(IReactorTime, reactor)
```

Comment:
Just asking. Wouldn't this do the job... as the explicit casting will tell mypy that the result is IReactorType ?

```suggestion
        self.clock = cast(IReactorTime, reactor)
```

Reply:
yup